server/routes/prisonIncentiveLevels.ts

```diff
@@ -25,13 +25,19 @@ export default function routes(router: Router): Router {
 
     const { id: prisonId, name: prisonName } = res.locals.user.activeCaseload
     const canEdit = hasRole(res)
-    const prisonIncentiveLevels = await incentivesApi.getPrisonIncentiveLevels(prisonId)
+    const [incentiveLevels, prisonIncentiveLevels] = await Promise.all([
+      incentivesApi.getIncentiveLevels(),
+      incentivesApi.getPrisonIncentiveLevels(prisonId),
+    ])
+    const activeLevelCodes = new Set(prisonIncentiveLevels.map(prisonIncentiveLevel => prisonIncentiveLevel.levelCode))
+    const inactiveIncentiveLevels = incentiveLevels.filter(incentiveLevel => !activeLevelCodes.has(incentiveLevel.code))
```

Comment:
I'm wondering if I'm missing something in this few lines of code.

The `getIncentiveLevels()` method seem to return only the active ones by default, the method call above doesn't pass `withInactive = true` so this seems to be the case.

However here the code seems to imply it's trying to find the levels inactive at the prison by doing the difference between all the levels (including inactive) and the ones active at the prison...but `incentiveLevels` wouldn't include the inactive in this case right?

Sorry if I'm missing something fundamental here.